pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pysolarfocus"
-version = "3.0.0"
+version = "0.99.0"
```

Comment:
Won't this cause version conflicts?
Or are you going to change this to 3.0.1 or  something similar?

Reply:
Yes, the plan is to continue with 3.x I used to lower version number to avoid causing issues with version >= notations